youtube_dlc/downloader/external.py

```diff
@@ -202,85 +237,34 @@ class Aria2cFD(ExternalFD):
 
     def _make_cmd(self, tmpfilename, info_dict):
         cmd = [self.exe, '-c']
-        # cmd += self._configuration_args([
-        #    '--min-split-size', '1M', '--max-connection-per-server', '4'])
-        # dn = os.path.dirname(tmpfilename)
-        # if dn:
-        #     cmd += ['--dir', dn]
+        dn = os.path.dirname(tmpfilename)
         if 'url_list' not in info_dict:
             cmd += ['--out', tmpfilename]
+        cmd += self._configuration_args(['-x16', '-j16', '-s16'])
+        if dn:
+            cmd += ['--dir', dn]
         if info_dict.get('http_headers') is not None:
             for key, val in info_dict['http_headers'].items():
                 cmd += ['--header', '%s: %s' % (key, val)]
         cmd += self._option('--interface', 'source_address')
         cmd += self._option('--all-proxy', 'proxy')
         cmd += self._bool_option('--check-certificate', 'nocheckcertificate', 'false', 'true', '=')
         cmd += self._bool_option('--remote-time', 'updatetime', 'true', 'false', '=')
+        cmd += ['--console-log-level', 'warn', '--file-allocation', 'none', '--uri-selector', 'inorder', '--download-result', 'hide']
```

Comment:
These should be added only when using `url_list`. For normal downloads, it is up to the user to decide the value they want for log-level, file-allocation etc. For fragmented downloads it is justified to override the user-provided args with these since the output would be VERY large otherwise

I do think these are good defaults (except `--file-allocation`), but they should not override user-provided options (put it above/inside `_configuration_args`) for normal downloads.